fastly/custom_tls_configuration.go

```diff
@@ -0,0 +1,151 @@
+package fastly
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/google/jsonapi"
+)
+
+// CustomTLSConfiguration represents a TLS configuration response from the Fastly API.
+type CustomTLSConfiguration struct {
+	ID            string       `jsonapi:"primary,tls_configuration"`
+	DNSRecords    []*DNSRecord `jsonapi:"relation,dns_records,dns_record"`
+	CreatedAt     *time.Time   `jsonapi:"attr,created_at,iso8601"`
```

Comment:
Please move metadata fields to the bottom of the struct definition.

Reply:
moved.